.env.baklava

```diff
@@ -71,6 +73,30 @@ AZURE_ORACLE_CENTRALUS_ADDRESS_AZURE_KEY_VAULTS=0x412ebe7859e9aa71ff5ce4038596f6
 AZURE_ORACLE_CENTRALUS_FULL_NODES_COUNT=2
 AZURE_ORACLE_CENTRALUS_FULL_NODES_DISK_SIZE=30
 
+# ---- Forno ----
+
+# ---- General Forno ----
+
+# A list of every context that forno will use.
```

Comment:
One question for my own curiosity, k8s forno works independently of the vm testnet b/c it relies on full nodes deployments and not tx/tx-private?

Reply:
Exactly